.prettierrc.js

```diff
@@ -3,4 +3,5 @@ module.exports = {
   trailingComma: 'all',
   tabWidth: 2,
   printWidth: 120,
+  endOfLine: 'crlf',
```

Comment:
'LF' would be preferable but's ok. Everyone should set up an editor adequately to make less mess.

Reply:
Well I changed that because it was CRLF at the begining. Prettier by default converts to LF, but after that git was yelling that there is a change in the end of line format and i couldn't figure out what to do with it.